src/content/content_manager.cc

```diff
@@ -315,8 +323,9 @@ void ContentManager::shutdown()
             log_debug("AutoScanDir {}", i);
             std::shared_ptr<AutoscanDirectory> dir = autoscan_inotify->get(i);
             if (dir != nullptr) {
-                if (fs::is_directory(dir->getLocation())) {
-                    auto t = getLastWriteTime(dir->getLocation());
+                auto dirEnt = fs::directory_entry(dir->getLocation());
+                if (dirEnt.is_directory()) {
+                    auto t = to_time_t(dirEnt.last_write_time());
                     dir->setCurrentLMT(dir->getLocation(), t);
```

Comment:
setCurrentLMT should probably be taking std::chrono::seconds or something.

Reply:
Good point. The focus was on filesystem operations. I put it on my todo list.